src/idl_gen_php.cpp

```diff
@@ -129,43 +131,51 @@ class PhpGenerator : public BaseGenerator {
   }
 
   // End enum code.
-  static void EndEnum(std::string *code_ptr) {
+  void EndEnum(std::string *code_ptr) {
     std::string &code = *code_ptr;
     code += "}\n";
   }
 
   // Initialize a new struct or table from existing data.
-  static void NewRootTypeFromBuffer(const StructDef &struct_def,
-                                    std::string *code_ptr) {
+  void NewRootTypeFromBuffer(const StructDef &struct_def,
+                             std::string *code_ptr) {
     std::string &code = *code_ptr;
 
+    std::string fqName = WrapInNameSpace(struct_def);
```

Comment:
use snake_case not camelCase, here and elsewhere.

Reply:
Fixed